helix-view/Cargo.toml

```diff
@@ -50,6 +50,7 @@ toml = "0.8"
 log = "~0.4"
 
 parking_lot = "0.12.2"
+nonempty = { version = "0.10.0", features = ["serde", "serialize"] }
```

Comment:
We try to keep the set of dependencies reasonably small and I don't think this one pulls its weight. It could be useful to have a wrapper around `Box<[T]>` that has serde implementations in the `helix-stdx` crate for uses like this but we don't need the whole NonEmpty dep

Reply:
Okay great, let me do that